spec/requests/api/go_cardless/webhooks_spec.rb

```diff
@@ -51,7 +51,11 @@
           ).to include({
             go_cardless_id: 'payment_ID_123',
             page_id: page.id,
-            amount: 100
+            amount: 100,
+            charge_date: Date.today,
```

Comment:
if you find out that the event does come back with a charge date, should update this spec as well